src/legacy/core_plugins/kibana/public/dashboard/dashboard_state_manager.ts

```diff
@@ -473,8 +473,8 @@ export class DashboardStateManager {
    */
   public applyFilters(query: Query, filters: esFilters.Filter[]) {
     this.appState.query = query;
-    this.savedDashboard.searchSource.setField('query', query);
-    this.savedDashboard.searchSource.setField('filter', filters);
+    this.savedDashboard.searchSource!.setField('query', query);
```

Comment:
It looks like those are not necessary because search source will always be available on a saved dashboard: https://github.com/elastic/kibana/pull/52850/files#diff-f96c4b2bd6a25826492a597d739afa82R39

Reply:
Thx! done!